Runtime/Scripts/Plugins/AnimationPointer/MaterialPropertiesRemapper.cs

```diff
@@ -0,0 +1,485 @@
+                GltfSecondaryPropertyName =
+                    $"pbrMetallicRoughness/baseColorTexture/extensions/{ExtTextureTransformExtensionFactory.EXTENSION_NAME}/{ExtTextureTransformExtensionFactory.OFFSET}",
+                ExtensionName = ExtTextureTransformExtensionFactory.EXTENSION_NAME,
+            };
+            AddMap(baseColorTexture);
+
+            var emissiveFactor = new MaterialPointerPropertyMap(MaterialPointerPropertyMap.PropertyTypeOption.SRGBColor)
+            {
+                PropertyNames = new[] { "_EmissionColor", "_EmissiveFactor", "emissiveFactor" },
+                GltfPropertyName = "emissiveFactor",
+                GltfSecondaryPropertyName =
+                    $"extensions/{KHR_materials_emissive_strength_Factory.EXTENSION_NAME}/{nameof(KHR_materials_emissive_strength.emissiveStrength)}",
+                ExtensionName = KHR_materials_emissive_strength_Factory.EXTENSION_NAME,
+                CombinePrimaryAndSecondaryOnImport = true,
+                ExportKeepColorAlpha = false,
+                CombinePrimaryAndSecondaryDataFunction = (primary, secondary) =>
+                {
+                    float strength = (secondary != null && secondary.Length > 0) ? secondary[0] : 1f;
+                    var result = new float[primary.Length];
+                    for (int i = 0; i < 3; i++)
+                        result[i] = primary[i] * strength;
+                    if (result.Length == 4)
+                        result[3] = primary[3];
+                    
+                    Color color = result.Length == 3 ? new Color(result[0], result[1], result[2]) : new Color(result[0], result[1], result[2], result[3]);
+                    color = color.gamma;
+                    result[0] = color.r;
+                    result[1] = color.g;
+                    result[2] = color.b;
+                    if (result.Length == 4)
+                        result[3] = color.a;
+                    return result;
+                }
+            };
+            AddMap(emissiveFactor);
+
+            var emissiveTexture = new MaterialPointerPropertyMap(MaterialPointerPropertyMap.PropertyTypeOption.TextureTransform)
+            {
+                PropertyNames = new[] { "_EmissionMap_ST", "_EmissiveTexture_ST", "emissiveTexture_ST" },
+                GltfPropertyName =
+                    $"emissiveTexture/extensions/{ExtTextureTransformExtensionFactory.EXTENSION_NAME}/{ExtTextureTransformExtensionFactory.SCALE}",
+                GltfSecondaryPropertyName =
+                    $"emissiveTexture/extensions/{ExtTextureTransformExtensionFactory.EXTENSION_NAME}/{ExtTextureTransformExtensionFactory.OFFSET}",
+                ExtensionName = ExtTextureTransformExtensionFactory.EXTENSION_NAME
+            };
+            AddMap(emissiveTexture);
+
+            /*var roughnessTex = new MaterialPointerPropertyMap
```

Comment:
Is this intentionally missing?

Reply:
Updated it in: 15542c7e0755edec44ffb5f1c42f6aba13c57fce